### PR TITLE
blob microinteraction

### LINKED ISSUE #65: Blob microinteraction
**Is your feature request related to a problem? Please describe.**
This will let users know if the ingredient has been selected

**Describe the solution you'd like**
Microinteraction on click

### PR DESCRIPTION
**Pull request by gonzalespaulb**
## Changes
_List Changes Introduced by this PR_
1. Add blob microinteraction when clicked

## Purpose
Helps user know if ingredient has been clicked

## Approach
Styled components keyframes

Closes #65

### CODE REVIEW

**Review by Lap343 — APPROVED** [review]
Very nice job. I am enlightened.

File: src/components/Blob.tsx [hunk 1]
```diff
@@ -24,16 +25,60 @@ const H1Container = styled.div`
...
+  animation-duration: 0.5s;
+  animation-fill-mode: forwards;
+  animation-iteration-count: 1;
+`;
+
+const Blob: FC<BlobProps> = ({ ingredientName, addIngredient, selectedIngredients}) => {
+  const [blobClicked, setBlobClicked] = useState<Boolean>(false);
+
+  useEffect(() => {
+    setBlobClicked(selectedIngredients.includes(ingredientName));
```
Comment (connielion) [inline src/components/Blob.tsx:73]: Sick!

**Review by connielion — APPROVED** [review]
Awesome!!!

**Review by JorgeLVilla — APPROVED** [review]
Very interested to see this on the app!

File: src/components/Blob.tsx (hunk 1 above)
Comment (gonzalespaulb) [reply src/components/Blob.tsx:73]: Courtesy of Keyslayer

File: src/components/Blob.tsx [hunk 2]
```diff
@@ -24,16 +25,60 @@ const H1Container = styled.div`
...
+const BlobInitial = styled.div<BlobInitialProps>`
+  position: absolute;
+  z-index: 10;
+  background-color: #ef8080;
+  width: 100%;
+  height 100%;
+  transform: scaleY(0.1) scaleX(0.1);
+  border-radius: 15px;
+  opacity: 0;
+  animation-name: ${props => props.blobClicked ? blobIn : ``};
```
Comment (dannysanchez559) [inline src/components/Blob.tsx:63]: nice.....

**Review by dannysanchez559 — COMMENTED** [review]
thanks for the info

**Pull request merged**